src/components/RemoveTodo/RemoveTodo.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { TODOSTYPE } from '../../api/interface';
+import { removeSelectedTodo } from '../../store/todosList';
+import './RemoveTodo.scss';
+
+type RemoveType = {
+  todo: TODOSTYPE;
+  setTodoId: Function;
+  setUpdateList: Function;
+};
+
+export const RemoveTodo: React.FC<RemoveType> = ({ todo, setTodoId, setUpdateList }) => {
+  const [changedTodo, setChangedTodo] = useState<TODOSTYPE>(todo);
+  const dispatch = useDispatch();
+
+  const handleChangedTodo = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value) {
+      setChangedTodo({
+        ...changedTodo,
+        title: event.target.value,
+      });
+    }
+  };
+
+  const handleSaveChangedTodo = () => {
+    dispatch(removeSelectedTodo(changedTodo));
+    setUpdateList(
+      (prevList: TODOSTYPE[]) => prevList.map(
+        (item: TODOSTYPE) => (item.id === changedTodo.id ? changedTodo : item),
```

Comment:
```suggestion
        (item) => (item.id === changedTodo.id ? changedTodo : item),
```
Тип должен унаследоваться от 29 строки